### PR TITLE
fix 404 immediately after logging in

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/server/index.js [hunk 1]
```diff
@@ -75,7 +75,9 @@ app.prepare()
         server.use(keycloakClient.middleware());
 
         logger.info("adding the /login handler");
-        server.get("/login", keycloakClient.protect());
+        server.get("/login", keycloakClient.protect(), (req, res) => {
+            res.redirect("/");
+        });
```
Comment (slr71) [inline src/server/index.js:80]: Apparently we were never hitting the `/login` URL directly before. It must have always been `/login/dashboard` or something like that.

**Comment by psarando** [comment]
> :shipit: immediately

Just a play on the PR title... not actually issuing orders 😜

**Comment by slr71** [comment]
Thanks for the review!

**Comment by slr71** [comment]
Er...reviews!

**Pull request merged**